spec/services/builders/events/pull_request_spec.rb

```diff
@@ -50,6 +50,22 @@
           .not_to change { Events::PullRequest.count }
       end
     end
+
+    context 'when multiple hooks for the same pull request are sent at the same time' do
+      let(:threads) { [] }
+
+      it 'does not raise a validation error' do
+        expect {
+          2.times do
+            threads << Thread.new do
```

Comment:
interesting.. I'm wondering if this test fails consistently without the changes or not 🤔 Did you test running this test without the code changes?

Reply:
Yes, first I created the test and made sure that it fails and when I change the code it does not failed 😄

Reply:
nice!